test/roomsAndSpeakers.js

```diff
@@ -0,0 +1,357 @@
+
+'use strict';
+
+const assert = require('chai').assert;
+let generator = require('../src/backend/generator.js');
+let app = require('../src/app');
+let webdriver = require('selenium-webdriver');
+let speakerPage = require('../src/selenium/speakerPage.js');
+let roomPage = require('../src/selenium/roomPage.js');
+
+describe('app', () => {
+  describe('run', () => {
+    it('should run app', () => {
+      const expressApp = app.getApp();
+      assert.equal(expressApp.get('port'), (process.env.PORT || 5000));
+    });
+  });
+});
+
+describe('generate', function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/app/triplez/open-event-webapp/pullRequest?prid=3226522)